doc/networking.md

```diff
@@ -162,6 +162,65 @@ message PingPong {
 }
 ```
 
+
+## Orchestrator To Redeemer
+
+*Applicable when running a ticket redemption service by using the `-redeemer` flag*
+
+### gRPC `QueueTicket`: `Ticket` -> `QueueTicketRes`
+
+`QueueTicket` is a unary RPC method that is called by an Orchestrator to send a winning ticket to the `Redeemer`. The request message has following format: 
+
+```
+message Ticket {
+    TicketParams ticket_params = 1;
+    bytes sender = 2;
+    TicketExpirationParams expiration_params = 3;
+    TicketSenderParams sender_params = 4;
+    bytes recipient_rand = 5;
+}
+```
+
+The `Redeemer` will send back an empty message, `QueueTicketRes` on success as well as a `200 OK` status. Upon error a `500 Internal Server Error` error code will be returned along side the error. 
+
+### gRPC `MaxFloat`: `MaxFloatReq` -> `MaxFloatUpdate`
+
+`MaxFloat` is a unary RPC method that is called by an Orchestrator when it requires the max float for a sender but no local cache is available. It's request takes in a sender's ethereum address:
+
+```
+message MaxFloatReq {
+    bytes sender = 1;
+}
+```
+
+The server will respond with the `MaxFloat` for that `sender` in raw bytes format:
+
+```
+message MaxFloatUpdate {
+    bytes max_float= 2;
```

Comment:
Let's update this to match the definition that was updated as a part of this [comment](https://github.com/livepeer/go-livepeer/pull/1513#discussion_r441216728).

Reply:
fixed in fcb86dd293a40d0a61916d875394afeb32a80dd5